src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -1082,4 +1082,87 @@ This version used in: {2}</value>
   <data name="Inline_variable_declaration" xml:space="preserve">
     <value>Inline variable declaration</value>
   </data>
+  <data name="All_lowercase" xml:space="preserve">
+    <value>All lowercase</value>
+  </data>
+  <data name="All_uppercase" xml:space="preserve">
+    <value>All uppercase</value>
+  </data>
+  <data name="Begins_with_0" xml:space="preserve">
+    <value>Begins with '{0}'</value>
+  </data>
+  <data name="Begins_with_0_camel_case" xml:space="preserve">
+    <value>Begins with '{0}'; camel case</value>
```

Comment:
Hrmm..  should we instead generate something like a comma/semicolon separated list dynamically?  I'm not sure how that would work with loc.  but i worry that you'd have to create resource strings for these sort of combined rules.